test-configs.yaml

```
@@ -56,6 +56,11 @@ file_systems:
     type: debian
     ramdisk: 'buster-v4l2/20200821.0/{arch}/rootfs.cpio.gz'
 
+  debian_buster-ltp_nfs:
+    type: debian
+    ramdisk: 'buster-ltp/20200819/{arch}/initrd.cpio.gz'
+    nfs: 'buster-ltp/20200819/{arch}/full.rootfs.tar.xz'
```

If you add this:
```suggestion
    nfs: 'buster-ltp/20200819/{arch}/full.rootfs.tar.xz'
    root_type: nfs
```
then you shouldn't need to specify the template file pattern in the test plan definition.